### ISSUE TITLE
No more highlighting after email2discord login

### ISSUE THREAD

**Issue by aayojezuzchrezt** [LAION-AI/Open-Assistant#1268]
Firstly I signed up via email, but then I added discord account while being signed it, now my name isn't being highlighted in scoreboard and I don't get scores for now
![image](https://user-images.githubusercontent.com/122345469/217068395-7d7ddcc6-acd8-4074-b484-05d856f1c24a.png)

**Comment by andreaskoepf** [comment]
Currently discord and mail accounts are not linked. We originally had an issue for this https://github.com/LAION-AI/Open-Assistant/issues/106 but it was closed.

@fozziethebeat Do we plan to support linking of email/discord accounts in the near future?

**Comment by aayojezuzchrezt** [comment]
Now there is a clone of my account, and all statistics scores here (1st pic)
![image](https://user-images.githubusercontent.com/122345469/217099558-f1e16d8b-fe7e-4f78-a758-ed63c87ac2e7.png)
![image](https://user-images.githubusercontent.com/122345469/217099655-203621d2-0506-4cf1-b2bc-e57e59f4183b.png)

**Comment by olliestanley** [comment]
> Now there is a clone of my account, and all statistics scores here (1st pic) ![image](https://user-images.githubusercontent.com/122345469/217099558-f1e16d8b-fe7e-4f78-a758-ed63c87ac2e7.png) ![image](https://user-images.githubusercontent.com/122345469/217099655-203621d2-0506-4cf1-b2bc-e57e59f4183b.png)

As Andreas mentioned above, when you logged in with Discord you didn't link your existing account, you made a new account. Currently we don't have account linking

**Comment by aayojezuzchrezt** [comment]
> > Now there is a clone of my account, and all statistics scores here
> 
> As Andreas mentioned above, when you logged in with Discord you didn't link your existing account, you made a new account. Currently we don't have account linking

Yes, got it, but now I can't get access to the first account in any way. Nor logging via email nor discord get 8th place (2nd pic) highlighted, but both work for 71st place, so what happened to the 1st email account?

**Comment by fozziethebeat** [comment]
You're first email account should still be there and you should be able to login via that method.

Does your discord use the same email as the email address you used originally?

**Comment by aayojezuzchrezt** [comment]
> You're first email account should still be there and you should be able to login via that method.
That's strange, 'cause it's no more, now both methods goes into the new account, and old one is inaccessible
> Does your discord use the same email as the email address you used originally?
Yes
